### DISCUSSION TITLE
Icone Homekit

### DISCUSSION THREAD

**Discussion by fabiomusicman** [indomus/forum#1493] [HUB: Home Assistant]
Ho degli shelly flashati con firmware tasmota ed integrati in homeassistant. Vorrei che in HomeKit (app casa) avere la possibilità di cambiare icona mettendo quella della porta garage o cancello ecc ecc. 
Nel menù tasmota c'è la possibilità di impostare il "modo"? se si come si deve fare?
purtroppo posso solo scegliere di usare LUCE VENTILATORE INTERRUTTORE. 
Grazie

**Comment by snakuzzo** [comment]
Non è una cosa che fai su Tasmota. Puoi impostare l'icona lato HA, ma non avrà nulla a che vedere con Homekit.
Se vuoi dai un'occhiata [QUI](https://www.home-assistant.io/integrations/homekit/) e vedi se cambia qualcosa impostando `type` o `device_class`
